test/e2e/operator_test.go

```diff
@@ -2057,6 +2059,226 @@ func TestNetworkLoadBalancer(t *testing.T) {
 	}
 }
 
+// TestAWSELBConnectionIdleTimeout verifies that the AWS ELB connection-idle
```

Comment:
Do you also need to add/change coverage for the GCP changes you made?

Reply:
I think `TestInternalLoadBalancerGlobalAccessGCP` already provides sufficient coverage.